gradle/versions.gradle

```diff
@@ -157,6 +157,8 @@ dependencyManagement {
 
     dependency 'org.fusesource.jansi:jansi:2.4.1'
 
+    dependency 'org.hibernate.validator:hibernate-validator:8.0.1.Final'
```

Comment:
surprised to see this, can we narrow this dependency down to just jakarta?

Reply:
jakarta is only the API, we need an implementation and it is provided by `hibernate-validator`